### PR TITLE
Eden's Gate all language

### PR DESCRIPTION
**Pull request by VanSlanzar**
Added and remove missing translations all language, but it's not completed. It need add missing translations (ja and ko).

### CODE REVIEW

**Comment by VanSlanzar** [comment]
- Here are the captures of the missing translations, in zip.
[ja.zip](https://github.com/quisquous/cactbot/files/4605574/ja.zip)
[ko.zip](https://github.com/quisquous/cactbot/files/4605576/ko.zip)

File: ui/raidboss/data/05-shb/raid/e1s.js [hunk 1]
```diff
@@ -492,273 +492,274 @@
...
-        'Vice And Virtue! (T)': '恶习与美德！(坦克)',
-        'Vice And Virtue! (D)': '恶习与美德！(DPS)',
-        'Vice And Virtue! (H)': '恶习与美德！(奶妈)',
-        'Vice And Virtue (T)': '恶习与美德(坦克)',
         'Vice And Virtue (D)': '恶习与美德(DPS)',
         'Vice And Virtue (H)': '恶习与美德(奶妈)',
+        'Vice And Virtue (T)': '恶习与美德(坦克)',
+        'Vice And Virtue! (D)': '恶习与美德！(DPS)',
+        'Vice And Virtue! (H)': '恶习与美德！(奶妈)',
+        'Vice And Virtue! (T)': '恶习与美德！(坦克)',
```
Comment (VanSlanzar) [inline ui/raidboss/data/05-shb/raid/e1s.js:689]: I think there are 3 doubles. Which ones do you want to keep?

**Comment by Jaehyuk-Lee** [comment]
is `goofs` thing actually used?
I always ignored them.

**Comment by VanSlanzar** [comment]
goofs ? sorry, inot understand.

**Comment by Jaehyuk-Lee** [comment]
e1s.js has goofs variable. I don't know what's that for

**Comment by VanSlanzar** [comment]
Oh, I got it. I think it's just for fun.

**Comment by VanSlanzar** [comment]
I did not look at what was the untranslated line.
As this generates a missing translation, I did not delete the line (missing_true_ko). I think it can be deleted.

**Comment by Akurosia** [comment]
yes goofs is used but as @VanSlanzar already wrote its just for fun when the long cutscene plays mid fight

**Comment by VanSlanzar** [comment]
thank you for your clarification. @Akurosia

**Comment by VanSlanzar** [comment]
I think everything looks good. Only the translations indicated will be missing. They can be corrected in this request, otherwise it will be in another.

File: ui/raidboss/data/05-shb/raid/e1s.js (hunk 1 above)
Comment (quisquous) [reply ui/raidboss/data/05-shb/raid/e1s.js:689]: I'm not sure what you mean about doubles.  The exclamation point is there to (subtly?) indicate that it's the powered up version vs the normal version.

Comment (VanSlanzar) [reply ui/raidboss/data/05-shb/raid/e1s.js:689]: Ah I understand. It's like French, except that I just put the punctuation aside, I didn't have to create more. I did not think about it. thank you for enlightening me.

File: ui/raidboss/data/05-shb/raid/e3s.js [hunk 2]
```diff
@@ -115,8 +115,8 @@
       alertText: function(data, matches) {
         if (data.me == matches.target) {
           return {
-            en: 'Tank Buster on YOU',
-            de: 'Tankbuster auf DIR',
+            en: 'Tank buster on YOU',
```
Comment (quisquous) [inline ui/raidboss/data/05-shb/raid/e3s.js:118]: The previous was fine.  I'm not sure why you are changing this.

File: ui/raidboss/data/05-shb/raid/e3s.js [hunk 3]
```diff
@@ -125,9 +125,9 @@
         }
         if (data.role == 'healer') {
           return {
-            en: 'Tank Busters',
-            de: 'Tank Buster',
-            fr: 'Tank busters',
+            en: 'Tank buster',
```
Comment (quisquous) [inline ui/raidboss/data/05-shb/raid/e3s.js:128]: There are two tank busters, so it is Tank Buster*s* plural.

File: ui/raidboss/data/05-shb/raid/e2n.js [hunk 4]
```diff
@@ -48,9 +48,9 @@
       },
       suppressSeconds: 1,
       infoText: {
-        en: 'tank busters',
-        de: 'tank buster',
-        fr: 'Tank busters',
+        en: 'Tank buster',
```
Comment (quisquous) [inline ui/raidboss/data/05-shb/raid/e2n.js:51]: Why did you change this?

File: ui/raidboss/data/05-shb/raid/e2s.js [hunk 5]
```diff
@@ -115,9 +115,9 @@
         return data.role == 'healer';
       },
       infoText: {
-        en: 'tank busters',
+        en: 'Tank buster',
```
Comment (quisquous) [inline ui/raidboss/data/05-shb/raid/e2s.js:118]: Please don't change this either.

File: ui/raidboss/data/05-shb/raid/e4s.js [hunk 6]
```diff
@@ -16,9 +16,9 @@
         return data.role == 'healer' || data.role == 'tank';
       },
       alertText: {
-        en: 'Tank Busters',
+        en: 'Tank buster',
```
Comment (quisquous) [inline ui/raidboss/data/05-shb/raid/e4s.js:19]: Please don't change this either.

File: ui/raidboss/data/05-shb/raid/e3s.js (hunk 2 above)
Comment (VanSlanzar) [reply ui/raidboss/data/05-shb/raid/e3s.js]: Oh , sorry.

File: ui/raidboss/data/05-shb/raid/e4s.js (hunk 6 above)
Comment (VanSlanzar) [reply ui/raidboss/data/05-shb/raid/e4s.js]: it's good.

File: ui/raidboss/data/05-shb/raid/e2s.js (hunk 5 above)
Comment (VanSlanzar) [reply ui/raidboss/data/05-shb/raid/e2s.js]: it's good.

Comment (VanSlanzar) [reply ui/raidboss/data/05-shb/raid/e2s.js]: it's good.

File: ui/raidboss/data/05-shb/raid/e2n.js (hunk 4 above)
Comment (VanSlanzar) [reply ui/raidboss/data/05-shb/raid/e2n.js]: it's good

File: ui/raidboss/data/05-shb/raid/e3s.js (hunk 3 above)
Comment (VanSlanzar) [reply ui/raidboss/data/05-shb/raid/e3s.js]: Sorry for this topic, I remembered a discussion. #1329 For Tank Buster or Busters. that's why I changed it, but I forgot the capital letters. Everything is corrected.

**Comment by VanSlanzar** [comment]
> In the future, if you want to sort the order of all of the translations, can you submit that as a separate pull request where the only thing you change is the ordering? It's hard to tell what's changed vs what's been reordered when it is all in the same pull request.

Oh sorry, of course.

**Comment by maiko-tan-coding** [comment]
> * Here are the captures of the missing translations, in zip.
>   [ja.zip](https://github.com/quisquous/cactbot/files/4605574/ja.zip) @maiko-tan-coding
>   [ko.zip](https://github.com/quisquous/cactbot/files/4605576/ko.zip) @Jaehyuk-Lee

Thank you for mentioning me, but I am busy these days, so I couldn't contribute. Sorry.

@r4ha Do you have time to fill those Japanese translations?
r4haさんお邪魔してすみません、このエデン覚醒編のトリガーを翻訳してもいいですか？

**Comment by Akurosia** [comment]
did't check all files directly but made you sure to remove uneccassary translations in all files? otherwise translating them now would be a waste of resources to translate stuf which should get thrown out anyway?

**Comment by VanSlanzar** [comment]
- I asked @quisquous  the question in MP, here is his answer.

I would say it's not really that important.  I'd say it'd be better to remove a line than translate one that isn't translated, but it's probably not worth removing lines that are unused from everywhere.
That just seems like something that could be done automatically with a tool.
And it's not just in the timeline, it's also about the triggers.

- As I often say, to translate into French everything is fine. But I leave it to you with your script which generates the answers in all languages for the rest. I would like to avoid making mistakes that would generate additional work for others. Understanding between us is not always easy, but my English is very poor, so I use google translate to make sure I understand better.

- the translations I requested are the missing lines that my reported python util / find missing .... so I think they are necessary.

- I try to help you my best, but if that poses problems, I would be satisfied to translate French and to sort the alphabetical order. Sorry for the inconvenience.

**Comment by r4ha** [comment]
@maiko-tan-coding すみません、よろしくお願いします！
